resources/benchmark.ts

```diff
@@ -95,7 +95,8 @@ function prepareBenchmarkProjects(
     const repoDir = path.join(tmpDir, hash);
     fs.rmSync(repoDir, { recursive: true, force: true });
     fs.mkdirSync(repoDir);
-    exec(`git archive "${hash}" | tar -xC "${repoDir}"`);
```

Comment:
I thought that Win10 supports `tar`, is it a problem with using pipe?
https://pureinfotech.com/tar-curl-windows-10/

Reply:
I guess so! I can update the description above.

Overall it's not clear to me why creating and then extracting a tar file is better than just creating a local clone, but if you feel we should create two different piping commands lines depending on the platform, that's fine as well.

As long as it works, I will be happy.